### PR TITLE
Remove Bento helpers, 3rd part

### PR DESCRIPTION
**Pull request by eduardoj**
Remove helpers only used in the old UI, from `s` till `w`, located under `src/api/app/helpers/webui/`.

### CODE REVIEW

**Comment by eduardoj** [comment]
[Remove bento helpers](https://trello.com/c/aaNWKJZf/1240-remove-bento-helpers)

File: src/api/spec/helpers/webui/webui_helper_spec.rb [hunk 1]
```diff
@@ -297,14 +164,6 @@
...
 
-  describe '#repo_status_icon' do
-    skip('Please add some tests')
-  end
-
-  describe '#tab' do
-    skip('Please add some tests')
-  end
-
   describe '#render_dialog' do
```
Comment (krauselukas) [inline src/api/spec/helpers/webui/webui_helper_spec.rb]: nitpick: I think the ```#render_dialog``` can be removed as well, at least I couldn't find something in the webui_helper that seemed to be related

Comment (eduardoj) [reply src/api/spec/helpers/webui/webui_helper_spec.rb]: Added. Thanks!

File: src/api/spec/helpers/webui/webui_helper_spec.rb [hunk 2]
```diff
@@ -63,27 +63,6 @@
     end
   end
 
-  describe '#repo_status_icon' do
```
Comment (dmarcoux) [inline src/api/spec/helpers/webui/webui_helper_spec.rb]: This helper is still present, so we should update the specs instead of removing them.

Comment (eduardoj) [reply src/api/spec/helpers/webui/webui_helper_spec.rb]: Bento helper `repo_status_icon` and its tests were removed in 1a0eabefd1aab039f61e43415eea1a578e904393. Newer helper `webui2_repo_status_icon` was renamed to `repo_status_icon` in ef35b3ab2b06d2c53bf8a4ec7495956372225f13. That is why "the helper is still present". Do we really need a test for actual renamed to `repo_status_icon` helper, that has this code?:
```
  def repo_status_icon(status)
    REPO_STATUS_ICONS[status] || 'eye'
  end
```

Comment (dmarcoux) [reply src/api/spec/helpers/webui/webui_helper_spec.rb]: It's true that it's pretty basic code... I guess it's fine.

**Pull request merged**